src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -32,11 +32,12 @@ public void loadSomeData() {
 					final Map venue = gson.fromJson(content, Map.class);
 
 					List<Map> placeCategories = (List<Map>) venue.get("categories");
+					
 					scanner.close();
+					count++;
 				} catch (final FileNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00108)